pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/rebalance/RebalanceSummaryResult.java

```diff
@@ -227,19 +224,14 @@ public void increaseNumServerParticipants(int numServers) {
     }
   }
 
+  @JsonInclude(JsonInclude.Include.NON_NULL)
```

Comment:
does this class level annotation ensure that fields of this class that are not null aren't included, or does it work for when this object itself is null?